gluonnlp/data/sampler.py

```diff
@@ -106,6 +116,9 @@ class FixedBucketSampler(Sampler):
         smaller buckets.
     shuffle : bool, default False
         Whether to shuffle the batches.
+    batch_type : str, default 'sentence'
```

Comment:
samplers don't really know what a word or sentence is. all it knows is samples. I think what's relevant here is whether to use scaleup_key or to use average bucket sample sizes. could you think of an argument name that reflects this? maybe use_bucket_avg?